Two tests are failing with a nuget problem. @ViktorHofer @ayakael Is the failure related to this change?

- Queue: Build Browser wasm Release AllSubsets_Mono_RuntimeTests
- Tests: `JIT.CodeGenBringUpTests` and `JIT.jit64`
- Log file: https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-77509-merge-435b1597369e479bb9/JIT.CodeGenBringUpTests/1/console.b29c28be.log?helixlogtype=result
Failure message:
```
/datadisks/disk1/work/9D6508C9/p/wasm-test-runner/WasmTestRunner.proj : error MSB4242: SDK Resolver Failure: "The SDK resolver "NuGetSdkResolver" failed while attempting to resolve the SDK "Microsoft.Build.NoTargets/1.0.53". Exception: "System.IO.IOException: The system cannot open the device or file specified. : 'NuGet-Migrations'
```

--- a/Directory.Build.targets
+++ b/Directory.Build.targets
@@ -11,6 +11,21 @@
   <Import Project="$(RepositoryEngineeringDir)liveBuilds.targets" />
   <Import Project="$(RepositoryEngineeringDir)python.targets" />
 
+  <!--
+  When .NET gets built from source, make the SDK aware there are bootstrap packages
+  for Microsoft.NETCore.App.Runtime.<rid> and Microsoft.NETCore.App.Crossgen2.<rid>.
+  -->
+  <ItemGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
+    <KnownFrameworkReference Update="@(KnownFrameworkReference->WithMetadataValue('Identity', 'Microsoft.NETCore.App')->WithMetadataValue('TargetFramework', '$(NetCoreAppCurrent)'))">
+      <RuntimePackRuntimeIdentifiers>$(PackageRID)</RuntimePackRuntimeIdentifiers>
+    </KnownFrameworkReference>
+    <KnownCrossgen2Pack Update="@(KnownCrossgen2Pack->WithMetadataValue('Identity', 'Microsoft.NETCore.App.Crossgen2')->WithMetadataValue('TargetFramework', '$(NetCoreAppCurrent)'))">
+      <Crossgen2RuntimeIdentifiers>$(PackageRID)</Crossgen2RuntimeIdentifiers>
+    </KnownCrossgen2Pack>
+    <!-- Avoid references to Microsoft.AspNetCore.App.Runtime.<rid> -->
+    <KnownFrameworkReference Remove="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
   <PropertyGroup>
     <!--
       Define this here (not just in Versions.props) because the SDK resets it

